test/runtests.jl

```diff
@@ -2,7 +2,7 @@ using RegistryCompatTools
 
 using Test
 
-const rexversion = "\\d+\\.\\d+\\.\\d+"
+const rexversion = "\\d+\\.\\d+\\.?\\d*"
```

Comment:
I relaxed this as the test was failing on 
```
" MosekTools@\e[92m0.15.1\e[39m \e[91m0.9\e[39m"
```
which only has a 2-digit second version umber